### DISCUSSION TITLE
Module Naming Changes

### DISCUSSION THREAD

**Discussion by critchie** [primefaces/community#3270] [PrimeNG]
I noticed in V18 the 'Module' suffix is dropped from a lot of the components.

Where it used to be `import { ToolbarModule } ...` it is now `import { Toolbar } ... `

Is this going to be the standard going forward? I noticed there are some components that still use the 'Module' suffix such as ButtonModule and OrganizationalChartModule.

**Comment by mikev10** [comment]
> I noticed in V18 the 'Module' suffix is dropped from a lot of the components.
> 
> Where it used to be `import { ToolbarModule } ...` it is now `import { Toolbar } ... `
> 
> Is this going to be the standard going forward? I noticed there are some components that still use the 'Module' suffix such as ButtonModule and OrganizationalChartModule.

I have the same question. After upgrading to v19, I noticed that some components now have "Module" added back to their names. This inconsistency makes it challenging to name the components correctly and requires renaming them during version upgrades. Additionally, I couldn’t find any documentation explaining the naming conventions for importing these components.
